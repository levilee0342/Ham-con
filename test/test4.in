-18753
-24728
255
-1225
-25979
-28183
-4541
2642
-1858
-23586
-15161
-21789
-12553
