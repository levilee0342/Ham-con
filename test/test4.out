2642
-28183